docs/pages/bare/overview.mdx

```diff
@@ -25,52 +26,186 @@ If you've ever written a native module for React Native, you'll be surprised how
+  title="Expo Router"
+  Icon={RouterLogo}
+  description="Expo Router is a file-based routing library that offers advantages such as organized navigation hierarchy, automatic deep linking support, and more."
+  href="/router/introduction/"
 />
+
+## Common questions
+
+<Collapsible summary="How long will it take to adopt Expo in my existing React Native project?">
+
+Adopting Expo doesn't have to be done in one step. You can start with the _quick wins_ and then move on to more complex parts. You can also pick and choose which features you want to adopt based on what is most helpful for your project.
+
+</Collapsible>
+
+<Collapsible summary="What will I gain from using Expo in my React Native app?">
+
+Adopting Expo tools in your existing React Native app will help you maintain long-term maintenance support, easier native upgrades, and a better developer experience.
+
+</Collapsible>
+
+<Collapsible summary="Who uses Expo?">
+
+Expo is used by top companies worldwide that serve millions of end users. For more information, see our [Expo showcase](https://expo.dev/customers).
+
+</Collapsible>
+
+<Collapsible summary="What impact will adopting Expo have on my app's size?">
+
+The `expo` package has a small footprint since it only includes a minimal set of modules required in every app with autolinking infrastructure and other Expo SDK libraries that are built-in.
+
+Including `expo` package in your app adds 1 MB to the final size of apps on app stores. This package has a marginal cost. For example, on Android, it only adds 150 Kib and the rest of the size comes from the language runtime (Kotlin).
+
+</Collapsible>
+
+<Collapsible summary="Why did React Native recommend using Expo?">
+
+Most React Native developers solve common problems when building an app, such as implementing navigation, accessing Native APIs, upgrading to new versions, and more. This requires using a specific set of tools and libraries to build and maintain your app &mdash; which means you are creating your own framework.
+
+Expo solves these problems by providing a set of primitives and helping you (the developer) to focus on building your app. It also offers tools to iterate faster in development. For more information, see [Why React Native recommends using a framework](https://reactnative.dev/blog/2024/06/25/use-a-framework-to-build-react-native-apps).
+
+</Collapsible>
+
+<Collapsible summary="Do I have to get rid of my native projects to use Expo?">
+
+By default, Expo projects created with `create-expo-app` use [Continuous Native Generation (CNG)](/workflow/continuous-native-generation/) and do not contain android and ios native directories. If you incrementally adopt Expo in your existing React Native app, you don't have to remove these directories. You can continue with `npx expo run:[android|ios]` as alternates to commands offered by `@react-native-community/cli` to compile your app locally and keep the configuration of your native projects.
```

Comment:
```suggestion
By default, Expo projects created with `create-expo-app` use [Continuous Native Generation (CNG)](/workflow/continuous-native-generation/) and do not contain android and ios native directories. If you incrementally adopt Expo in your existing React Native app, you don't have to remove these directories. You can use `npx expo run:[android|ios]` as an alternative to commands offered by `@react-native-community/cli` to compile your app locally and keep the configuration of your native projects.
```
Slightly fast-and-loose with pluralization rules, but I think reads out loud a little cleaner